src/constants.ts

```diff
@@ -1,5 +1,127 @@
+import path from 'path';
+
+/**
+ * The name of this project. Used to exclude this repo as a lintable project.
+ */
+const THIS_PROJECT_NAME = 'module-lint';
+
+/**
+ * The root directory of this project.
+ */
+const THIS_PROJECT_DIRECTORY_PATH = path.resolve(__dirname, '..');
+
 /**
  * The number of milliseconds in an hour, used to determine when to pull the
  * latest changes for previously cached repositories.
  */
 export const ONE_HOUR = 60 * 60 * 1000;
+
+/**
+ * Wherever the tool was run.
+ */
+export const WORKING_DIRECTORY_PATH = process.cwd();
```

Comment:
Not something I would describe as a "constant" :thinking: And only used in one place. Maybe better off inlining this.

Reply:
Fair enough, inlined here: [732fc82](https://github.com/MetaMask/module-lint/pull/33/commits/732fc82c3f2d5e5cbe9147de6cd168e5ed098137)